mmv1/third_party/terraform/utils/tagslocation_operation.go

```diff
@@ -0,0 +1,65 @@
+package google
```

Comment:
Can you please rename this file to tags_location_operation.go to follow the convention?